common/components/controllers/CorporateHackathonController.jsx

```diff
@@ -399,7 +402,6 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
             </div>
           </div>
         </div>
-        {this._renderContactSponsor()}
 
         <div className="corporate-sponsorship-saying corporate-bg-light col-12">
```

Comment:
This is an instance of `corporate-bg-light` to remove for the section background issue.